tachyon/math/polynomials/univariate/sparse_coefficients.h

```diff
@@ -22,6 +22,9 @@ namespace tachyon::math {
 template <typename F, size_t MaxDegree>
 class DenseCoefficients;
 
+// Class to represent polynomials with sparse coefficients.
+// In contrast to dense representations, this only stores non-zero terms.
+// This is efficient for polynomials where most of the degrees have zero.
```

Comment:
```suggestion
// SparseCoefficients class provides a representation for polynomials where
// only non-zero coefficients are stored. This is efficient for polynomials
// where most of the degrees have zero coefficients.
```
There's nothing wrong with your comment but I suggest this way to have a more kind of consistency.